functions/release_notes/main.py

```diff
@@ -0,0 +1,57 @@
+import logging
+import os
+
+import flask
+import functions_framework
+
+from bugbug import generative_model_tool
+from bugbug.tools.release_notes import ReleaseNotesCommitsSelector
+from bugbug.utils import get_secret
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+os.environ["OPENAI_API_KEY"] = get_secret("OPENAI_API_KEY")
+
+llm_cache = {
+    "llm_name": None,
+    "llm": None,
+    "chunk_size": None,
+    "selector": None,
+}
+
+
+@functions_framework.http
+def handle_release_notes(request: flask.Request):
+    if request.method != "GET":
+        return "Only GET requests are allowed", 405
+
+    version = request.args.get("version")
+    if not version:
+        return "Missing 'version' query parameter", 400
+
+    llm_name = request.args.get("llm", "openai")
+    chunk_size = int(request.args.get("chunk_size", 100))
```

Comment:
I do not think we need to get the model name and chunk size from the request. Is there a specific reason for doing that?